spec/spec-helper.coffee

```diff
@@ -24,7 +24,7 @@ TextEditorElement = require '../src/text-editor-element'
 TokenizedBuffer = require '../src/tokenized-buffer'
 TextEditorComponent = require '../src/text-editor-component'
 pathwatcher = require 'pathwatcher'
-clipboard = require 'clipboard'
+clipboard = require "../src/safe-clipboard"
```

Comment:
Super minor, but typically single quotes are used for requires since they almost never have interpolated values in them